lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -316,5 +316,124 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_RELATEDPERSON_CREATE ||= {
+
+      'resourceType': 'RelatedPerson',
+      'extension': [
+        {
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/related-person-encounter',
+          'valueReference': {
+            'reference': 'Encounter/2467924'
+          }
+        },
+        {
+          'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/relationship-level',
+          'valueCodeableConcept': {
+            'coding': [{
+              'system': 'http://hl7.org/fhir/resource-types',
+              'code': 'Encounter'
+            }]
```

Comment:
nit: sometime the array bracket is on a new line sometimes not

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/commit/c5a51de325f428be24ed8e8c510ddcd737ea96c3#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R205